extension/js/contentScript.js

```diff
@@ -0,0 +1,20 @@
+$( document ).ready( function () {
+	// Inject page script into the DOM
+	injectScript(chrome.extension.getURL('js/pageScript.js'), 'body');
+
+	/**
+	 * injectScript - Inject internal script to available access to the `window`
+	 *
+	 * @param  {type} file_path Local path of the internal script.
+	 * @param  {type} tag The tag as string, where the script will be append (default: 'body').
+	 * @see    {@link http://stackoverflow.com/questions/20499994/access-window-variable-from-content-script}
+	 */
+	function injectScript(file_path, tag) {
+		var node = document.getElementsByTagName(tag)[0];
+		var script = document.createElement('script');
+		script.setAttribute('type', 'text/javascript');
+		script.setAttribute('src', file_path);
+		node.appendChild(script);
+		return script;
+	}
+} );
```

Comment:
The first and last line of this file has MediaWiki-style spacing between parentheses, and the other lines do not. Also I think eslint would want `filePath` instead of `file_path`, etc. I'm sure you know, but can run `./node_modules/.bin/eslint --fix extension/js/contentScript.js` to fix some of these issues automatically.

Reply:
I should add contentScript.js to the tested files in .eslint....